lib/testing/src/lib/core/pages/form/widgets/displayValueWidget.ts

```diff
@@ -32,4 +33,12 @@ export class DisplayValueWidget {
         return this.formFields.getFieldValue(fieldId, this.inputLocator);
     }
 
+    async isDisplayValueWidgetVisible(fieldId): Promise<void> {
```

Comment:
missing type for fieldId in both methods of this page

Reply:
added type for fieldId